crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -13,8 +13,8 @@ The library adds hardware support for selected cryptographic algorithms.
 
 Integration with Mbed TLS
 =========================
-The nrf_cc3xx_mbedcrypto library provides low-level integration towards selected versions of Mbed TLS deliverables.
-The APIs expressed in this library use the alternative implementation abstraction layer inside Mbed TLS for selected modules.
+The nrf_cc3xx_mbedcrypto library provides low-level integration with Mbed TLS.
```

Comment:
The extra statement on "selected versions" is due to the fact that we need a matching APIs, configurations, mbed TLS replacement files and Cmake logic to make it work

The library is by no means directly pluggable into any specifc Mbed TLS version. Maybe we need to add a new sentence instead

Reply:
Fixed